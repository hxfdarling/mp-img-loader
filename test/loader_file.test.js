/* eslint-disable
  prefer-destructuring,
*/
import webpack from '@webpack-contrib/test-utils';

describe('Loader', () => {
  test('file-loader', async () => {
    const config = {
      loader: {
        test: /\.(png|jpg|gif)$/,
        options: {},
      },
    };

    const stats = await webpack('file.js', config);
    const { source } = stats.toJson().modules[0];
    expect(source).toMatchSnapshot();
  });
});
