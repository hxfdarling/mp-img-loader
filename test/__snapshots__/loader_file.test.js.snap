// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader file-loader 1`] = `"module.exports = __webpack_public_path__ + \\"d32b2f66e51bbb729790dd3f1af13d7e.jpg\\";"`;
